src/tools/ceph_objectstore_tool.cc

```diff
@@ -3406,6 +3406,9 @@ int main(int argc, char **argv)
       "Threshold (in seconds) to consider omap listing slow (for op=list-slow-omap)")
     ("dump-data-dir", po::value<string>(&dump_data_dir),
      "Directory to dump object data (for op=dump-export)")
+    ("get-attrs", po::value<string>(&key_oid), "Get the object's attributes")
+    ("set-attrs", po::value<string>(&key_oid), "Set an object’s attributes")
+    ("rm-attrs", po::value<string>(&key_oid), "Remove an object’s attributes")
```

Comment:
Hey @Suyashd999, it looks good to me. Minor suggestion, perhaps you can add parentheses around each s since both commands seem to work.

```suggestion
    ("get-attr(s)", po::value<string>(&key_oid), "Get the object's attributes")
    ("set-attr(s)", po::value<string>(&key_oid), "Set an object’s attributes")
    ("rm-attr(s)", po::value<string>(&key_oid), "Remove an object’s attributes")
```